dockerfile/avalanche/Dockerfile

```diff
@@ -117,15 +114,25 @@ LABEL org.opencontainers.image.source="https://github.com/strangelove-ventures/h
 
 WORKDIR /bin
 
-# Install ln (for making hard links), rm (for cleanup), mv, mkdir amd vi from full busybox image (will be deleted, only needed for image assembly)
-COPY --from=busybox-full /bin/ln /bin/rm /bin/mv /bin/mkdir /bin/vi ./
-
 # Install minimal busybox image as shell binary (will create hardlinks for the rest of the binaries to this data)
 COPY --from=infra-toolkit /busybox/busybox /bin/sh
 
 # Install jq
 COPY --from=infra-toolkit /usr/local/bin/jq /bin/
 
+# Install rm
```

Comment:
we should install the new bins using hardlinks to the busybox binary instead of installing bins separately. See the comment below `# Add hard links for read-only utils`. We can remove `read-only` from that comment, and then also need to remove the `#  Remove write utils` step.